index.html

```diff
@@ -1831,8 +1831,92 @@ <h3>Verify Proof</h3>
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Let <var>vmIdentifier</var> be set to
+<var>proof</var>.<var>verificationMethod</var>.
+          </li>
+          <li>
+Let <var>vmPurpose</var> be set to <var>proof</var>.<var>proofPurpose</var>.
+          </li>
+          <li>
+If <var>vmIdentifier</var> is not a valid URL, an
+`INVALID_VERIFICATION_METHOD_URL` error MUST be raised.
+          </li>
+          <li>
+Let <var>controllerDocumentUrl</var> be the result of parsing
+<var>vmIdentifier</var> according to the rules of the URL scheme and extracting
+the primary resource identifier (without the fragment identifier).
+          </li>
+          <li>
+Let <var>vmFragment</var> be the result of parsing <var>vmIdentifier</var>
+according to the rules of the URL scheme and extracting the secondary
+resource identifier (the fragment identifier).
+          </li>
+          <li>
+Let <var>controllerDocument</var> be the result of dereferencing
+<var>controllerDocumentUrl</var>, according to the rules
+of the URL scheme and using the supplied <var>options</var>.
+          </li>
+          <li>
+If <var>controllerDocument</var>.<var>id</var> does not match the
+<var>controllerDocumentUrl</var>, a `INVALID_CONTROLLER_DOCUMENT_ID` error MUST
+be raised.
+          </li>
+          <li>
+If <var>controllerDocument</var> is not a valid <a>controller document</a>,
+an `INVALID_CONTROLLER_DOCUMENT` error MUST be raised.
+          </li>
+          <li>
+Let <var>verificationMethod</var> be the result of dereferencing the
+<var>vmFragment</var> from the <var>controllerDocument</var> according to the
+rules of the associated URL scheme and using the supplied <var>options</var>.
+          </li>
+          <li>
+If <var>verificationMethod</var> is not a valid <a>verification method</a>,
+an `INVALID_VERIFICATION_METHOD` error MUST be raised.
+          </li>
+          <li>
+If <var>verificationMethod</var> is not associated with the array of
+<var>vmPurpose</var>s in the <var>controllerDocument</var>, either by reference
+(URL) or by value (object), an `INVALID_PROOF_PURPOSE_FOR_VERIFICATION_METHOD`
+error MUST be raised.
+          </li>
+          <li>
+Return <var>verificationMethod</var> as the
+<strong>verification method</strong>.
+          </li>
+        </ol>
 
+        <p>
+The following example provides a minimum conformant
+<a>controller document</a> containing a minimum conformant
+<a>verification method</a> as required by the algorithm in this section:
+        </p>
+
+        <pre class="example" title="Minimum conformant controller document">
+{
+  "id": "https://controller.example/123",
+  "verificationMethod": [{
+    "id": "https://controller.example/123#key-456",
+    "type": "ExampleVerificationMethodType",
+    "controller": "https://controller.example/123",
+    <span class="comment">// public cryptographic material goes here</span>
+  }],
+  "authentication": ["#key-456"]
+}
+        </pre>
```

Comment:
The line breaks in this `<pre>` element are not preserved in the preview. The doc should be checked after this PR is merged, to ensure that the line breaks are preserved there.

Reply:
I just checked on my local copy, the formatting survives... this is probably an issue w/ the PR-Preview tool.